src/common/pageTitle/pageTitle.module.scss

```diff
@@ -0,0 +1,10 @@
+@import 'colours';
+@import 'spacings';
+
+.pageTitle {
+  color: $gray-500;
+  font-size: 1.25em;
```

Comment:
We have a partial for `font-sizes` that can be used here.
```SCSS
@import 'fonts';

.element {
  font-size: font-size(16);
}
```

Reply:
Indeed, but I'll use Text instead so this will not be needed